contrib/lang/ess/packages.el

```diff
@@ -140,11 +139,6 @@ not play nicely with autoloads"
 
 (when (configuration-layer/layer-usedp 'auto-completion)
```

Comment:
This condition is not needed anymore, the `post-init` function will be executed only if there exist a `init` function for company (which mean company is used).
This condition was only for `init-company-ess`.